### ISSUE TITLE
Calculations not saving

### ISSUE THREAD

**Issue by thiagobones** [AndreasReitberger/3D-Print-Cost-Calculator-App#4]
**Describe the bug**
after you produces 2 calculations, the app is not saving any new one. It will not clear the old information and will not save it.

**To Reproduce**
1. Go to calculator
2. Fill the details
3. Click on calculate
4. Go back to “calculations”
5. Check the list of saved calculations

**Expected behavior**
The calculations should be saved there but they are not. After the second one, theres just no more calculations saved

**Smartphone (please complete the following information):**
 - Device: iPhone 13 pro max
 - OS: iOS 16.1.1

**Comment by AndreasReitberger** [comment]
Hi,
This behavior is intended. The yellow note states out that you are working on a selected calculation.
To start a new one, simply click this yellow message to start with a fresh calculation.

![image](https://user-images.githubusercontent.com/2952443/208962976-12bd7c85-4906-4d4a-8669-4158d6b32ae2.png)

![image](https://user-images.githubusercontent.com/2952443/208963050-2e142024-753c-45a9-a6a6-4faaa0adcf35.png)

Hope this helps. If you still think there is a bug, please let me know.

Best,
Andreas

**Comment by thiagobones** [comment]
Hi,I did that but it still didnt save a new one. I’ll send you a video i madeEnviado do meu iPhoneEm 21 de dez. de 2022, à(s) 14:07, Andreas ***@***.***> escreveu:﻿
Hi,
This behavior is intended. The yellow note states out that you are working on a selected calculation.
To start a new one, simply click this yellow message to start with a fresh calculation.


Hope this helps. If you still think there is a bug, please let me know.
Best,
Andreas

**Comment by AndreasReitberger** [comment]
Yes that would help a lot. Also, please check the Event Manager if there is any error logged.
Have you tried the latest beta over TestFlight yet? I tried to reproduce it with that version, however it worked as expected.

**Comment by thiagobones** [comment]
I didn’t try that yet… how can i test it? I’ll try to reproduce it and let you knowEnviado do meu iPhoneEm 22 de dez. de 2022, à(s) 01:51, Andreas ***@***.***> escreveu:﻿
Yes that would help a lot. Also, please check the Event Manager if there is any error logged.
Have you tried the latest beta over TestFlight yet? I tried to reproduce it with that version, however it worked as expected.

**Comment by AndreasReitberger** [comment]
Just visit this link to enroll to the TestFlight progam:

https://testflight.apple.com/join/2OEBhYvr

Once all steps are done, you are able to download version 1.2.7, which is currently available for testing.

Best,
Andreas

**Comment by thiagobones** [comment]
Just did that! Thanks!I’ll try it out and let you know if the problem persists Enviado do meu iPhoneEm 22 de dez. de 2022, à(s) 15:31, Andreas ***@***.***> escreveu:﻿
Just visit this link to enroll to the TestFlight progam:
https://testflight.apple.com/join/2OEBhYvr
Once all steps are done, you are able to download version 1.2.7, which is currently available for testing.
Best,
Andreas
